starport/templates/query/stargate/x/{{moduleName}}/keeper/grpc_query_{{queryName}}.go.plush

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	"context"
+
+    "<%= ModulePath %>/x/<%= ModuleName %>/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k Keeper) <%= title(QueryName) %>(goCtx context.Context,  req *types.Query<%= title(QueryName) %>Request) (*types.Query<%= title(QueryName) %>Response, error) {
+	if req == nil {
+        return nil, status.Error(codes.InvalidArgument, "invalid request")
+    }
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+    // TODO: Process the query
```

Comment:
Can you elaborate on this?

Reply:
This is the placeholder to tell the user to put their own logic to process the query

Do you have other suggestion?